addons/modules/stringtable.xml

```diff
@@ -0,0 +1,245 @@
+        </Key>
+        <Key ID="STR_CBA_Modules_DefendModuleDescription">
+            <English>Sync to leader of group to defend a parsed location</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_Patrol">
+            <English>Patrol</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_PatrolCenterType">
+            <English>Patrol Center Type</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_PatrolCenterType_Desc)">
+            <English>Set what kind of object is being passed as the center point</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_PatrolPosition">
+            <English>Center Point</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_PatrolPosition_Desc">
+            <English>Enter an array with brackets or name without quotes</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_PatrolRadius">
+            <English>Patrol Radius</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_PatrolRadius_Desc">
+            <English>The distance to patrol from the center point</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_WaypointCount">
+            <English>Waypoint Count</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_WaypointCount_Desc">
+            <English>The amount of waypoints to create</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_WaypointType">
+            <English>Waypoint Type</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_WaypointType_Desc">
+            <English>The type of waypoint to be used</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_Move">
+            <English>Move</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_SAD">
+            <English>Seek and Destroy</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_Loiter">
+            <English>Loiter</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_Behaviour">
+            <English>Behaviour</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_Behaviour_Desc">
+            <English>The behaviour of the group</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_Careless">
+            <English>Careless</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_Safe">
+            <English>Safe</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_Aware">
+            <English>Aware</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_Combat">
+            <English>Combat</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_Stealth">
+            <English>Stealth</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_CombatMode">
+            <English>Combat Mode</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_CombatMode_Desc">
+            <English>How fast the group is to engage</English>
+        </Key>
+        <Key ID="STR_CBA_Modules_CombatYellow">
+            <English>Yellow(Fire at will)</English>
```

Comment:
Add a space just before the first parenthesis, ie `Yellow (Fire at will)` so as to improve readability. 

Same with the others below.